earthpy/plot.py

```diff
@@ -407,7 +407,15 @@ def make_col_list(unique_vals, nclasses=None, cmap=None):
     -------
     >>> import numpy as np
     >>> import earthpy.plot as ep
-
+    >>> import matplotlib.pyplot as plt
```

Comment:
@mirob9363  i made some small changes to this to see if it fixes the build. it should